app/src/main/kotlin/jp/co/yumemi/android/code_check/NetworkRepository.kt

```
@@ -0,0 +1,58 @@
+        try {
+            val response: HttpResponse =
+                client.get("https://api.github.com/search/repositories") {
+                    header("Accept", "application/vnd.github.v3+json")
+                    parameter("q", inputText)
+                }
+
+            val jsonBody = JSONObject(response.readText())
+            val jsonItems = jsonBody.optJSONArray("items") ?: return emptyList()
+
+            return parseRepositoryItems(jsonItems)
+        } catch (e: JSONException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)